src/transformers/models/biogpt/configuration_biogpt.py

```diff
@@ -0,0 +1,138 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Team and Microsoft Research AI4Science All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" BioGPT model configuration"""
+
+from ...configuration_utils import PretrainedConfig
+from ...utils import logging
+
+
+logger = logging.get_logger(__name__)
+
+BIOGPT_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    "kamalkraj/biogpt": "https://huggingface.co/kamalkraj/biogpt/resolve/main/config.json",
```

Comment:
Checkpoints will need to be transferred to the microsoft org before we merge.

Reply:
Let me know once the checkpoints are transferred, I can make the changes.
Thanks